src/pages/auth/signUp/index.tsx

```diff
@@ -34,9 +45,41 @@ const SignUpPage = () => {
 
   const { handleSubmit } = methods;
 
-  const onSubmit: SubmitHandler<SignUpDataType> = (data) => {
-    const mbti = `${data.EI}${data.NS}${data.FT}${data.JP}`;
-    console.log(mbti);
+  // 회원가입 mutate
+  const { mutate } = useMutation({
+    mutationFn: signUp,
+    onSuccess: () => {
+      alert('회원가입에 성공했어요.');
+      navigate(AUTH_PATH.LOGIN);
+    },
+    onError: () => {
+      alert('회원가입에 실패했어요.');
+    },
+  });
+
+  const onSubmit: SubmitHandler<SignUpFormDataType> = (data) => {
+    const {
+      EI,
+      NS,
+      FT,
+      JP,
+      'confirm-password': _,
+      certno,
+      ...filteredData
+    } = data;
+
+    const formData = {
+      ...filteredData,
+      authToken,
+      mbti: `${EI}${NS}${FT}${JP}`,
+      imageUrl: '',
+      location: {
+        latitude: latitude,
+        longitude: longitude,
+      },
```

Comment:
```suggestion
      location: {
        latitude,
        longitude,
      },
```

이렇게 줄여서 작성하는건 어떨까요? 저는 이렇게 작성하는게 코드도 간단하고 어떤 값을 value로 가지는지 덜 헷갈릴 것 같습니다..!

Reply:
수정했습니닷 👍